sweep/sweeper.go

```diff
@@ -19,6 +19,66 @@ type Sweeper struct {
 	Lnd *lndclient.LndServices
 }
 
+// CreateUnsignedTaprootKeySpendSweepTx creates a taproot htlc sweep tx using
+// keyspend. Returns the raw unsigned txn and the sighash or an error.
+func (s *Sweeper) CreateUnsignedTaprootKeySpendSweepTx(
```

Comment:
Ideally for new functions we should always add explicit unit tests.

Reply:
Added unit test coverage (separate commit).